backend/service/k8s/deployment.go

```diff
@@ -87,6 +87,62 @@ func ProtoForDeployment(cluster string, deployment *appsv1.Deployment) *k8sapiv1
 	return k8sDeployment
 }
 
+func ProcessObjProbe(objProbe *v1.Probe) *k8sapiv1.Probe {
```

Comment:
Lets add some test for this function, also do we want this function to be public? i think will probably want to keep it private.

Reply:
Remove public method and add this method to testing